autockt/envs/ngspice_ledro_d_fc45.py

```
@@ -0,0 +1,496 @@
+    )
+
+    def __init__(self, env_config):
+        self.multi_goal = env_config.get("multi_goal", False)
+        self.generalize = env_config.get("generalize", False)
+        num_valid = env_config.get("num_valid", 50)
+        self.specs_save = env_config.get("save_specs", False)
+        self.valid = env_config.get("run_valid", False)
+
+        self.env_steps = 0
+        with open(LEDRO_D_FC45.CIR_YAML, "r") as f:
+            yaml_data = yaml.load(f, OrderedDictYAMLLoader)
```

Ruff (0.12.2)

Probable use of unsafe loader `OrderedDictYAMLLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)